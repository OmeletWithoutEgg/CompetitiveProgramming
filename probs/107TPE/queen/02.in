1866
-79188 -92718
70335 66126
61879 52948
-92467 10489
-46430 21958
15939 -47572
64766 -99159
8830 -6834
27507 -5978
-89410 -31926
90880 -66050
-42958 -59953
23871 -79116
-84925 -1622
-86376 45289
-5956 -70634
-8651 65382
6689 -40664
58465 -97780
44350 11149
23518 -14155
80649 -87862
49031 -68657
-88984 95951
-74379 91195
8653 55729
-46740 -32980
47955 -67265
-6911 -75675
84327 -29381
40433 21550
-34034 -24407
-15958 38737
-76972 99809
-67127 -82150
80893 47824
-34535 -26530
-4028 -51710
-485 -2805
21589 -465
99610 365
-24660 37318
-59726 -59632
87537 83114
6160 65936
-9271 -68933
-19921 45292
-62488 -94211
-75369 93249
13602 52145
-25650 -10063
-73682 96666
4764 518
92583 -73364
12370 -5212
-36150 -82980
62937 -83173
-4773 79361
-76760 -51557
82002 58294
56407 -3581
42218 97120
55661 -40480
44632 87697
-12059 -46640
5287 -51344
-45194 22874
42618 60574
-75053 -53460
-30310 -96390
62661 16613
99964 74444
-48818 -64017
62773 69174
-30534 -85791
-83917 72548
68661 -36404
24834 -43528
-15572 -741
18251 -97250
61577 -9428
81102 -16151
49624 -5325
-29174 24230
-63793 -32575
-50665 2154
10061 -46270
-84732 -30106
10143 90692
-37518 -46608
75161 -42963
5779 94768
23482 18770
13344 75783
-82568 -20127
-10770 57036
59005 9411
75280 -41820
77466 -62801
61307 12209
-52748 90808
87709 -48967
-94127 39113
-64490 18411
87939 -72270
13796 6433
-15555 -19501
-798 63527
-7667 -18805
-11464 -1283
70977 55797
89737 -43710
33471 -89783
83686 16415
-65156 26894
-17883 -19738
8805 79618
2770 54668
-54056 29303
82245 -32863
38624 -74123
-12472 34640
-54070 25517
-11394 41520
4773 -76800
62593 -37950
-58847 84650
25407 76107
57453 -19388
512 54703
-93579 96280
-68116 98239
42809 -48370
-65637 -44461
41495 26127
4775 -68738
87523 27711
-98883 47456
-53448 -73356
-19463 -28799
24811 52600
69515 56159
62805 5119
-76917 -3865
4052 12710
44785 -9197
13352 -65930
68750 9817
35833 -15034
7331 -65852
74058 40884
-55348 66252
85117 2969
40047 15408
78337 -87368
-90123 80004
98438 -16033
-75640 81813
46199 84600
-25834 -72659
92468 90247
6900 -11207
87670 64581
11848 -36683
54838 -77622
50814 95021
-70327 -56623
-3186 31891
-89890 -5474
60642 26099
-80575 92885
35784 12233
-64768 88622
70926 -13920
-56315 -11659
-23682 74570
-29258 24029
94214 -35194
-13760 -32289
-17979 -1377
68451 45608
-88453 65
38406 90341
63914 1808
-58949 3689
4361 91937
-16229 53062
-23096 54820
-8828 87613
85718 -67034
-58086 10755
-60387 74124
18504 45575
-91288 73430
-44611 39917
4904 37556
-10646 -40488
19364 95143
-60621 -54112
-71368 -78098
21368 -7716
17680 -28902
-81 -17153
-94658 32353
42991 -80215
-34668 -96893
-96887 -82289
-7981 -39528
19129 80286
-48239 60331
-34061 -70324
65611 42676
-20019 80710
-69916 -52399
63931 -36804
-4651 -75942
-83824 52642
20896 -20472
21943 33714
33571 75412
-95490 4890
-46310 66925
-14876 29335
-40230 -25489
25418 -25646
-19255 -66252
-94260 -96109
8334 -38388
-7252 -73461
78998 59700
-30044 -4383
-71985 87039
61709 -65574
62240 74222
-7535 -19360
-80551 -83342
96830 11738
61762 35078
16634 72670
49092 -17178
-79579 -79350
86015 63100
65546 -85105
1415 13341
-25215 92006
-31943 40258
-61880 95497
-77690 -18291
-60066 -8279
-36201 27839
-8399 -45885
-36274 24323
23682 3239
-14463 -35209
65500 -5108
-93176 61600
53284 -1767
43975 52534
58537 74163
28278 55144
82535 -47953
72269 -29051
-91382 34329
-20 -64537
55747 36899
76218 5437
56120 -28390
16978 -25207
-41976 -8561
8143 42745
61685 78244
-60426 -71293
55321 -74337
-4611 -91036
54176 -40942
54623 27994
-51232 79936
-57992 -77067
76566 77155
98968 45761
-62158 91132
-5591 32075
-4494 83878
-96294 -91658
-69068 -82275
-12434 90645
-47010 -7795
-53232 68111
95812 12386
59709 -8900
13313 33090
85172 49642
95963 53073
65260 -48609
24235 -65108
-62085 -12659
-53920 -35050
23413 97892
9444 97641
6249 -53971
7322 -25474
-73544 69100
-73984 17420
63585 -664
-92041 93433
-5390 -33018
-32128 93762
-64598 38749
44833 91443
-99058 -94939
84378 28628
71005 31386
-23756 89453
25858 28096
95050 95913
-56767 72635
30315 -3523
79298 99571
28091 -57676
93113 15436
-65769 -29414
-32976 -38127
4469 15171
89825 -915
57249 -75854
-7805 37623
-2142 67287
-65100 -84301
-71684 72158
-92598 36247
-24754 -70975
-3969 74776
-77396 43229
-89673 15398
42710 90123
-3277 76933
94987 -82045
-98489 15286
-5307 -61382
-52826 67118
47175 61610
-81343 -64070
-48290 -35679
-82498 -32477
36811 -19144
-14062 26089
-32594 62465
-67700 47730
89343 -40040
61889 -60646
-89553 -10347
32812 -24371
40845 -49466
45886 515
95699 -9611
40012 58695
66659 80862
-45121 7040
47060 -46877
-97113 73941
94563 -22490
64783 -27647
-1328 -91853
89047 -59238
-55485 45875
-29953 72902
62755 -2574
-67380 72525
-29819 -98600
-20810 -1919
1993 -4552
-62007 3038
55667 34778
-64468 -66911
62688 86389
99035 86032
-83530 25306
73466 -9486
-41866 -90860
51824 -72888
16586 83142
92106 99668
25061 81563
-34141 -57336
9054 -94691
15464 -15432
-95117 -57375
-54901 62089
-47895 11371
9090 -55336
26120 -64298
-64735 11670
41255 -34859
-39804 -70853
98934 -25556
36638 54385
-88140 66278
91554 -3750
3148 15330
-23158 89445
-72928 66498
-38372 -50602
-63153 48010
78168 -14824
-48772 35303
-10918 -41742
-53008 2937
-22613 -7955
28826 -48247
56940 71012
82043 87096
57444 -78624
3914 45024
-51007 90692
34164 -55514
-21903 85248
49095 -74951
36740 62703
-73996 -20185
19127 -20222
-52871 -82674
-97738 -76463
79744 -35742
-99893 16804
88816 86856
27186 48470
-54463 -89123
5949 -98580
15503 -41733
-42270 36694
83599 -5880
-64671 18190
16397 -26923
33107 -53125
-33666 19405
-99376 -48205
-13101 50057
-96789 92702
-78776 52651
19385 -6338
-86681 66788
1185 -32588
2377 -81197
38539 -59368
39273 81349
-52685 -34022
45310 -37893
-81812 29376
21940 54826
78824 -57666
-65823 -1432
72775 -55277
37876 -3433
-12676 72406
-68990 -63034
-83919 57128
56796 -1713
-25978 46772
51089 73980
-96786 78985
-53150 -56173
-22092 -5395
42706 1605
-47781 -93068
77758 -34563
-5222 -58898
40869 -66617
-62643 43903
-74821 88768
-72544 34830
95560 74497
-4275 68218
50127 19671
-93246 34995
53062 -35599
-70580 -63918
10478 20635
-73569 19023
98361 -37888
19693 44068
-82162 -83211
60249 15839
-11652 33290
20442 -19875
48470 85448
24018 70460
48071 -99685
83390 -99315
50433 -43705
-10924 -59153
-73390 -1217
20155 30271
-35010 47133
53791 80605
6750 19978
-4080 76730
72306 -75859
60934 48397
30426 -78790
57376 -32738
10036 -10069
-15579 -83270
13825 63784
7821 20863
-34404 6780
70486 -89542
63284 59471
84405 49058
82098 23476
-28945 -85330
69907 70618
-13183 -44378
9028 75610
79736 790
35267 16891
-74771 96515
-33517 -46347
12948 88280
30878 51607
-8392 -93382
-681 -23236
80804 31188
-78758 -30367
58841 -13165
39444 53527
8264 -3096
-66385 -7552
85764 18252
96503 13656
91353 10047
-90587 -7938
-73333 -40762
10590 99554
-14717 -61480
-54045 -9561
11617 -65947
-15341 -94378
-57677 16443
-74179 16745
-96511 11446
-17618 -21261
-89629 66929
-41985 -65568
-65323 -97712
40623 -85003
-5828 -44483
-2807 -49196
-806 -83567
88437 -70729
-96082 -67585
-43346 -44084
-32579 -18953
-35064 49564
93175 97735
-56487 584
-23053 60609
88915 9909
50254 -42704
-34108 57318
54303 54318
22618 45561
45890 64623
75740 84144
-62242 -38240
-2878 -99624
-1006 -69503
-34609 66020
75863 56070
-23105 -91731
-98899 42288
71143 64361
86865 -23382
-47537 -96480
91873 31124
47743 6533
-3905 33921
80475 -83559
-93642 -17834
-31427 73086
-37876 50270
83337 73114
83807 51574
21934 -62064
42360 37110
92904 -57908
70133 90513
73445 -82466
33414 44007
-97614 74267
30044 33646
58238 -1253
21141 74051
68366 49797
43887 264
44903 -50133
-4749 6261
26963 13812
-14616 -31754
79859 47936
97430 64482
1611 99682
61018 77064
64367 -97138
-96994 -90693
18414 -58918
99545 -73999
82804 12523
-30201 -86092
-78266 34345
72297 11790
86659 23047
-3000 28688
84392 96284
-23055 -63802
46840 -35218
57000 46181
-25427 -64466
8463 67413
-37029 -98237
86844 1366
52465 11551
-62153 19637
-72421 79002
62710 50462
-65330 51399
20588 -62483
33005 -89949
-11441 -39484
-51839 -57954
3282 -29364
47010 -31568
-60505 -82876
8280 18343
-34123 47865
-33090 -44569
-22184 -87075
56007 45816
-85269 -6158
51230 79966
-95070 44134
-98968 85670
30111 -80202
-65053 -82630
94048 14401
-37946 -75861
-55057 45142
-98562 27588
45428 81679
-82202 -69413
80728 79657
-68261 -70539
69665 59459
70789 2783
-4485 39591
76632 -24619
-86370 28949
-26398 -81244
49361 -89578
-83672 83237
-674 -49650
99902 39772
13207 17322
-69132 -2513
88164 23414
-24192 66871
28567 -78243
-33445 80515
-31791 -91601
24862 74663
-26633 25690
89993 18524
-35390 -32076
76243 -49787
15729 -68904
54684 51622
-4309 -34363
-57409 -7850
-1122 92619
-60823 -77066
-67330 -53173
64430 -64307
64446 -73507
-29383 -72275
-8267 61768
-85210 -75509
93815 98783
-70703 -83556
-57136 42771
-782 71991
54868 -9276
-22790 -18731
97525 39612
56322 33614
-66315 58985
34634 -43845
-77845 91325
-25852 -7742
10571 -85709
61760 -34424
-70636 -76044
46697 -33679
19190 97951
91557 -19906
39774 92348
-30351 -53104
-84971 60651
-22769 -96900
-18714 -20046
-14648 -2254
34532 91380
-71872 -653
34052 23438
-61586 -6264
-70051 -97147
-3428 13707
-61516 -61936
77059 -51381
10527 -82238
-6888 -98968
-99605 91911
-56900 -84437
50878 40696
77462 39197
-39704 -84161
-54358 55154
82615 35542
96178 -44851
-46655 -26333
-79853 68842
60149 34498
92023 31761
70892 29572
24070 -28894
67357 45519
34298 -45426
-10237 -56983
-61695 60772
99429 77436
-89070 -57363
-32693 -99316
-8731 52516
19345 78197
41871 -34715
16016 46852
-49526 49643
-3328 -70550
-49421 62976
80874 -75704
52454 76917
-20396 -66288
66066 41719
-44586 8545
-16296 -91818
3177 -75623
9524 -71713
-88658 10026
-85013 -89772
-12690 37074
21285 6854
-72289 32081
78255 -76352
-23389 13734
-43254 -40231
-35902 -92032
12202 69979
47480 -41556
-94665 34457
91365 8367
933 -77453
38460 26602
92817 43087
-68917 16865
-30293 -36638
92421 7415
-12872 43371
90509 90743
86008 58244
-50734 58226
-66281 23733
92734 -97166
56473 26724
42174 -84820
-64335 -90042
-11371 -54072
2279 14689
-99414 -54474
14800 64630
36680 -15779
-78917 63193
-34695 -40815
40785 -30700
-13247 96826
-48146 -45086
81879 -4661
-92839 -23988
21706 -86395
-89600 -91585
41574 87130
15103 65413
75598 -35874
68823 -30504
21180 -65142
-41101 -3729
-29732 88174
58390 -46786
-67148 -73509
49935 20556
-14288 67110
34038 -39603
-72208 19056
85145 -98197
-69888 -37156
6856 58025
-98394 -28421
-8817 24964
32032 56759
34133 76101
75263 -4837
-94922 46302
69854 -86340
29568 -13206
52503 -66788
-6114 -52608
88496 25096
-24309 75301
94097 -37238
56850 13864
66722 93351
-71949 87730
60829 -48616
14537 -13788
-99664 -52556
34988 30587
97457 49375
13897 -88898
38537 -54133
-21551 22013
-67853 -46208
20870 -83318
81141 -12730
-7035 -5921
-10538 -91717
-17354 96329
58963 9431
-19290 27631
50534 93008
83487 -60448
60266 -47577
-53404 -8052
-28103 32840
25674 -5702
-11435 30069
64821 88716
58577 -31310
52730 65811
40497 24302
78362 95588
-77236 -84485
27488 -20238
-74507 -20629
21127 82276
-97156 -93455
94415 -62780
-12767 -33863
39312 -40161
-85602 52038
95556 -45686
41521 44192
92883 33122
-77060 71178
-21748 -28627
-90606 -30385
-70134 -18267
-12325 -92871
35923 76296
3026 64250
-95972 4301
-37045 -36852
-20653 -73733
-88298 -71654
-96281 63928
-60226 -58699
-41989 -71573
63486 33186
15614 42700
88027 -67757
79376 60980
76429 25639
-39019 -73572
-37619 -87683
-38369 -12464
70727 -14961
-14802 -85083
89303 18759
-92289 99358
-2519 -20497
62354 -16394
44614 18862
81546 -74702
-79653 63151
92566 -58635
13107 -19376
-66875 -29309
-25403 48293
-35250 -33037
-73082 82052
-82857 -88823
-61074 18890
88594 -84050
-31965 -3512
82144 -62077
6561 83644
25087 35141
52853 93688
24991 -54967
-16498 85865
-45279 -65056
-65518 35951
81965 97940
46922 88959
938 10892
18730 -90820
-33868 85664
70014 -97688
-48248 -56811
65670 90556
-73822 -11814
-97957 72736
22664 96725
40926 98645
96279 -81942
-36596 -45302
92369 -57154
-19773 -39287
-50700 -62696
-66291 15331
-43364 -5162
17664 -62959
-82892 38710
40121 38182
-50697 -41296
-77555 82473
76119 38031
51023 -26172
30564 80629
-26053 18011
64297 -30428
53813 28732
-58320 -42228
-28922 -24218
85597 47212
-75770 98089
-53814 64437
28518 7135
-33155 -82484
15318 -26372
99140 82789
78475 11355
-46316 -19774
90570 84342
68424 91630
80627 -12438
81168 79576
-79131 -5395
7243 -95818
72666 -15702
-12088 -27846
1132 -18254
-1089 -10744
94888 -66145
-94128 -45234
-58910 37109
88041 -59531
-23581 40643
-4296 -22189
-18324 -5514
50087 81332
73630 -43501
48319 -90549
56909 -7420
-30825 56145
15459 93011
81362 37927
72700 17795
-14086 26533
83938 -99052
-78404 36428
-15046 -97116
61230 -93913
77035 22484
34974 90255
44054 -40763
-80844 -48748
85968 -79916
83389 31957
-18904 -29503
-7038 -95917
-68878 8348
96640 60269
25898 -47787
-204 -19245
-17990 47148
41099 -64598
51774 7165
48273 -68290
10298 54715
-15237 52137
-6052 -51878
65955 27272
-4235 81377
48362 -38523
93484 38720
20052 3398
94909 30102
-63566 42678
14958 -84937
-89275 -34926
66073 -24338
-43936 -60125
17783 16771
-91820 -28061
-40060 -42066
37664 39002
11973 -91888
1331 -29341
-13721 -63331
-36945 -80930
85477 -60643
79177 78336
-53474 -49102
18797 137
-11703 87608
71349 52495
95840 80882
227 -37623
75197 -33085
-86415 94773
-99084 79038
-2650 -16891
998 -12704
35773 58188
73345 -24866
-46231 -48588
-32290 -37295
-92214 -97983
-9902 26518
4530 -53935
-97962 54851
-77332 -72999
99991 -90709
80137 33499
8309 69983
77793 -29642
12573 -68521
-45888 95732
23117 -17904
79767 98538
28089 78088
-56202 77282
64574 -99122
-83279 -88577
80688 -59223
-50101 -59352
-79401 -364
-16822 -74922
-55120 89465
43937 53455
70376 80894
-18341 -31219
92454 220
65013 77921
-15588 55308
-90741 39443
2672 -20012
-72884 -74420
-58882 -64998
-12964 -83443
-11628 -44727
67504 -80668
73191 -5440
-38392 -96555
57135 -10422
43851 -66038
89476 28846
-249 -80845
86544 -46180
-94081 -20033
20030 90340
-7380 -32214
-59446 65516
-81421 22318
-2378 91409
-76773 -77957
1933 56810
-19778 99125
56744 46534
-22842 -35140
-19705 -74120
67573 -77224
-69072 99864
15236 -41469
2802 -76623
58605 -46522
-16590 64796
77387 -97448
-29294 5928
-68855 -69868
-45096 -34403
-92895 -54823
-35660 86266
-58906 97253
762 -81066
90041 97491
-93524 73591
-13667 62061
-93232 1689
-53708 -81362
-82056 13112
36386 -7602
26814 3323
-98335 -73590
85438 -93973
-75442 -37981
-6068 92434
-38841 -63463
77536 48373
-27692 -39157
-82939 -22715
8820 72393
88008 -68430
43632 -85513
20691 86376
82732 95785
11811 26752
411 -81755
-96378 -7592
95132 -95228
62130 -92971
97177 -18098
23613 79510
-97119 61141
-79941 -40815
-93193 48325
73041 68341
54017 66180
15074 14786
-82066 -94236
-31630 -58943
-74938 21269
-88610 -79361
32155 1744
4545 -92463
48852 -83246
94084 80550
29040 -2787
24987 43710
-93369 -65531
-36940 7627
52474 8933
-31406 -4060
-32416 -85520
27318 -57404
-90890 -42776
-25352 -82398
-92087 -1349
-47070 52847
-76071 25823
-68863 -93511
-78577 -43285
-21092 -10921
69833 51781
-24385 -68569
36019 -26323
14452 -52969
-6056 26742
-22304 70592
68255 -43928
10222 52318
74597 -77174
-42654 9156
22647 87411
14560 -80933
-83294 -27599
-16792 -9608
-18243 48841
62031 89556
44646 68960
88872 -58821
49942 -44866
-6061 95437
21885 54783
-55153 48739
-58575 -99640
-89260 65420
34389 41602
44610 17613
39909 -36064
-3110 -13278
70265 75779
-25614 45543
-20331 23362
96431 -81987
-44930 -87614
54766 63827
24602 21491
57430 2137
-21548 69536
27040 80880
-30188 99986
-83843 -2286
-55525 -24469
-10263 26189
40652 46921
-33602 34872
29886 27769
-46548 66822
-59224 -90658
31602 44441
-16155 -60308
-40038 13169
-5546 -18329
63919 4936
-56281 5962
-52562 -49192
-93755 -90580
-49767 37436
-62191 -52426
72182 46887
-77146 -52838
27933 2540
-43209 -84478
-13754 -274
-24126 -98104
84205 -84775
59449 -71834
56795 26568
-91488 -46713
51988 -51019
59363 -26542
98679 95938
85092 65827
-92202 -15370
30242 -43662
82637 -48151
84265 27115
91598 -23328
-77113 -22304
68475 44779
-44848 -3839
-85192 92292
57153 54917
72732 15429
-44938 65659
-19765 73542
95220 95694
-5522 -21718
93497 23465
26665 96776
78784 -7998
47943 -31582
-40703 22097
33420 -27300
18707 -16859
23370 31176
-40419 85636
74731 -66339
72634 -22883
-67458 81110
52623 -32449
36289 -55008
87073 -72353
-79162 48833
-11895 -78888
-50194 -6423
41420 71361
65475 56834
97173 69587
7908 68017
-73001 -27260
-71987 67395
18253 -67156
-82854 89733
-73176 11020
-2736 32343
30598 -51150
-74331 87615
14652 94300
7919 -66757
17698 51016
-84531 87641
-39798 -54612
98042 -25960
-97590 33493
65532 -72636
-32180 -77196
-87885 61424
-66725 53259
95564 -90479
49471 -15007
76051 4336
37520 -87337
63470 -83043
12695 98876
23754 -82442
35819 66055
60210 17960
-83113 -65247
-6132 -78801
39946 27399
98768 77774
-37528 -73974
88919 -65918
53833 -16509
52855 -70545
-41214 92715
76887 67158
65316 -21451
86549 -92039
-51800 -57179
59421 -69771
5168 -89052
96982 67778
-78920 55169
19236 -44865
-83076 -37901
-86397 78745
81040 98010
86880 -59281
-26687 -18181
69941 41451
14643 -89762
62253 87903
69501 -75659
-38119 25033
60695 36362
45316 41540
24839 2798
-82998 54437
-79389 -66723
-14207 54287
-25164 -77721
1627 42576
58691 -46362
-1688 -49553
59053 -27670
-5727 -92205
25749 34646
39936 7606
-34167 35692
35136 35841
-15908 90899
-34633 -22198
-57655 15483
24428 -24050
62061 70144
-22191 -16020
67037 91986
-55883 -99679
88907 82845
95630 -50966
79892 84722
-33135 54599
61360 -32423
14343 -71161
4502 -46906
-26135 15005
-56207 86275
-93292 60403
2114 -62736
-26315 -95822
-10407 91482
-81404 -49039
18016 -12214
67364 38544
97540 -85485
10139 90053
-79212 -21322
-89924 6382
94449 -74698
-93687 13262
37330 -9720
-18359 -63863
55781 -88132
41646 77120
-71176 88873
41196 67965
-59755 -24857
91843 -25612
44937 -82290
-83873 -95460
-75504 22808
6047 5709
-89718 -34614
-15148 -85709
-36084 49325
98000 -28660
50101 19436
54746 -22053
57674 55508
-33705 46444
-39198 -61139
5846 -72888
-44295 -79862
-43464 -83715
-4717 46581
-35433 -75280
-7033 44789
-51365 -20824
-65247 55867
-79561 77831
37205 30984
-82447 97915
-67739 33259
31432 76141
52660 9475
27260 -95847
-28894 -52758
9303 -21880
26717 96721
-18450 67219
70944 42183
40790 -74395
1148 67574
73573 -63989
-17327 -76012
86201 -3571
-11807 -82125
-47932 19255
-52743 -41361
49273 -60521
58384 69634
-42971 -65791
-96011 84501
21361 48308
55001 -6519
32801 -99779
-99874 -56380
94205 67549
97319 -79078
11819 14618
-2562 -10281
97359 82994
44481 51728
73914 -14180
15378 -38276
31513 -31074
30699 38807
19368 -83736
-42227 -36499
45220 38540
46475 81336
-13503 94487
50580 -6358
-93039 61041
39002 -62864
-51737 -53937
-49111 90668
-47715 37743
-93349 92191
61273 10986
-49980 -76687
-93665 -34729
-7873 -52052
-28969 -75394
68052 -20523
31843 41882
-10959 -25416
75927 -18502
-40186 40410
-103 69882
-93635 76571
-65492 19317
-94300 -78677
-82410 38635
-84139 8986
56160 -7999
-50637 12677
-19043 21832
41282 6621
84024 99547
27597 2059
41797 11855
-48742 57147
-86631 -74085
-32643 -14103
-63628 79877
-51163 90093
36469 43410
80206 -66393
-71523 -49153
7971 -29355
86220 -81708
491 10213
-85400 -73892
21896 -97559
50936 38932
73321 -27110
-83455 56590
75165 -66459
16504 77115
-34727 -75793
-30194 -63659
-24668 -59571
67713 39063
4790 -95358
-55496 95732
-42966 -37497
-70096 43904
20191 44109
76931 24501
15504 21562
91075 -16741
87955 -76333
-18661 -12535
7370 -86328
33254 40891
-53181 92725
-9077 35645
-54975 14497
1493 40924
-54197 44673
87821 -67127
-94268 -84329
-46695 11484
-44185 -22572
62777 56785
25644 30950
85936 -94032
-67940 4036
-25386 -38913
9808 67295
81665 82774
33581 -83079
24217 -36643
-94686 16347
-72941 -7369
-71521 99130
-35679 99243
-50796 56285
51374 83335
26492 91443
-37411 69949
7054 -19300
-79041 -65070
94721 -94718
-97003 87562
69897 33377
-56599 -62048
78387 68415
21055 37307
-99614 68735
90164 77190
67257 43016
90311 -97339
90576 -29066
46698 -23820
-13027 -18438
39307 -93706
77269 -67083
-29557 34667
-29215 -38670
-49148 -81903
-76681 -58923
10971 21299
73502 96899
39384 -52784
-39949 97766
74652 25325
-70284 86998
69120 -74832
-37395 51992
-60113 -86454
96675 15330
4615 51916
-75888 48028
92874 94854
-81195 80255
-92477 35461
-4916 99033
19437 -9838
4370 -27904
-70921 -15919
-29937 340
62809 86960
2566 1330
-90460 -72473
-18755 -40387
26354 -41839
18688 1147
-35137 60435
-61322 1225
-57025 23331
27892 66751
-45963 -82103
23971 -63872
64386 39746
-22665 -17444
-27345 26660
-73691 22333
-10393 83201
25405 97660
-86051 -14798
63180 -26909
68621 32547
-92083 -25805
46134 -13184
-79735 -99908
33343 -31740
96022 78453
-37216 -56740
-58488 1118
-93456 80549
-54 91519
18726 90642
-18072 2
-21761 -11078
13230 52832
73148 35862
-64865 -65190
60271 42180
25401 26244
91565 -81323
-19260 -7463
-48317 49446
-8090 -58437
23680 45441
72464 19947
35008 -64069
-15336 -62958
84913 -5841
51601 21496
62659 -1916
-49671 28867
96182 46581
-7831 14411
-53690 -79471
-92495 -48064
-42944 -67541
-1239 25098
-33225 57094
7479 37547
41078 76701
-26714 717
-65162 69542
-89021 -73698
-75441 69417
93599 1304
-50081 49959
73471 10317
-3229 68311
6953 40135
-51292 -38430
-91572 20092
78392 -27154
30325 76117
-36212 -97039
9320 81071
70072 50283
19131 -11960
98577 -73338
90211 28146
-89964 -92609
-31428 39903
-27676 94943
-48980 15297
-78156 39606
63136 45056
57281 -26232
84760 -67448
-95968 68032
89531 -21689
-21163 84080
-70456 53990
-31690 27548
-97259 -10782
12271 84137
85478 69224
85962 -32357
52128 13062
-94460 -57057
47566 -39907
-75341 -872
-25339 40283
72048 -59707
29610 -21268
-22476 55206
-57020 69976
38194 -65359
17206 -90180
-45349 77988
28962 40889
-28548 -14345
78289 -28956
62653 20108
-14384 49393
-52857 -87077
-32400 63409
-75010 45022
-66905 -50410
78724 -37352
-89441 99519
-67551 62804
-9090 -11753
46554 27206
8843 -17239
81906 -86659
-15526 66796
31094 -9671
62572 25975
-61496 -13755
-71070 89840
43152 -15513
-49124 29267
61972 -90757
-63016 39965
25860 70318
57420 -72541
-95997 60215
-67595 9247
-44609 -9149
-73089 48943
80210 -31731
-12286 -30385
-82160 63729
-38783 -4373
-26030 -35609
-24077 23371
5238 75353
-30365 -69196
92206 25764
1245 25958
-82783 13869
-83618 -17925
47176 84010
-96053 74638
-17472 -10108
43633 39038
8875 74137
58663 -72948
-13991 -58992
-14897 -26601
-24330 -41859
27213 -87339
63395 6913
-58262 -24377
-86112 -87876
25424 92061
-23134 47770
-61712 -80689
-95291 70531
-36418 -8848
88652 -91484
36736 78742
79795 33904
60581 -19573
-95107 99396
-61796 -82936
64576 -68209
-15585 -7340
75695 -629
-39748 99576
-94201 -85599
-50003 -62964
69844 16290
-27836 -60825
-79294 59495
-53604 -59374
52672 54777
2268 -6479
72219 10683
-62873 40626
89942 -81739
-38545 -41688
-47652 27418
43748 -90366
-88372 1635
-5643 -3744
-21753 22732
29571 96683
-28313 -60254
73184 78624
26468 -73497
-45464 -42576
70726 -73505
-84569 -45659
-92831 95902
-94935 -73845
93077 -85452
-93700 37276
37168 62709
92522 -57826
89359 97387
23612 90952
58959 2081
82415 53442
-4224 84051
94762 64475
-22148 78057
-3555 61567
-62649 87632
-44553 18803
-3776 11166
69465 2319
70421 28869
11959 73323
47069 44695
50690 56284
68133 83756
-66029 -96920
-1280 52483
-90324 18021
86007 -18893
41389 -76083
-4648 -61731
-38551 16793
-56053 19020
82327 -56957
83418 -88857
87919 -97908
46491 -83660
28964 36193
68856 -80912
78157 -45051
-64006 -74521
-32636 -91186
-25579 68677
-6934 65324
-93485 18169
-33513 -73933
38997 87322
37469 64512
-49915 68167
15296 97214
-33844 8882
-2395 -98998
-26616 73536
90611 -54227
53439 -62189
-28710 67584
-79395 56154
97659 30255
99708 -89345
-36208 52503
-99696 45348
9817 -41591
-79574 78745
-75800 28378
-42149 -9349
87238 -33223
56564 -53388
-71734 -96257
-21658 48536
-2960 -97890
72255 -96483
-36051 -38682
37396 -33014
67100 -49206
65205 78547
49056 -21805
12974 78744
51523 -65251
-5750 -10670
2574 -71694
-32897 -41303
-42688 -19852
-44398 73976
