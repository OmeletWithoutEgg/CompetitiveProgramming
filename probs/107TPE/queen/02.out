2320.38 -25799.98
