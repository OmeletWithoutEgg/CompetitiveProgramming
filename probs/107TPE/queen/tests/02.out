9814.00 -50293.00
